packages/server/src/rooms.ts

```diff
@@ -55,12 +55,12 @@ async function doJoinRoom(socket: Socket, store: Store, room: Room, player: Play
   if (room.players.length === MAX_ROOM_SIZE) {
     let state = store.getCurrentState(room.name);
     if (!state) {
-      const playerNames = room.players.map((player) => player.name);
+      const playerNames = room.players.map((player) => playerName);
```

Comment:
This doesn't make sense it will return twice the same username

Reply:
Correct, done.

Reply:
Still doesn't :-) what's the logic? You are mapping over a list and returning a list with the same username? Why. What are you trying to do? I really don't get it

Reply:
You can remove the map and return [playername, playername] directly. Plus player the parameter of the function is not used here we need some eslint in the server too. are you sure it's what you want?

Reply:
Ok I see you removed but for some reason I didn't see the changes. Nevermind